Sources/FaunaDB/Path.swift

```diff
@@ -27,6 +27,8 @@ extension Path: CustomStringConvertible {
     }
 }
 
+/// Represents a path segment to a field value.
```

Comment:
This is missing something, but I'm not sure what. I think maybe "a path segment pointing to a field's value" but I don't really like that too much either. I'll think of something